src/poller.ts

```diff
@@ -0,0 +1,97 @@
+import { DIPDeposit } from "./common";
+import {
+  Connection,
+  AccountChangeCallback,
+  PublicKey,
+  AccountInfo as solanaAccountInfo,
+  Context,
+  clusterApiUrl,
+} from "@solana/web3.js";
+import { AccountInfo, AccountLayout, u64 } from "@solana/spl-token";
+
+export class Poller {
+  cluster: string;
+  callback: (deposit: DIPDeposit) => void;
+  splToken: string;
+  premiumAsset: string;
+  expiration: number;
+  strike: number;
+  type: string;
+
+  constructor(cluster: string, callback: (deposit: DIPDeposit) => void) {
+    this.cluster = cluster;
+    this.callback = callback;
+    this.splToken = "BTC";
+    this.premiumAsset = "USDC";
+    this.expiration = Date.now();
+    this.strike = 25000;
```

Comment:
just for testing? why hardcoded?

Reply:
Just for testing. Will need to copy the code from the UI that lists out all the DIP